detection-rules/link_dynamics_form.yml

```diff
@@ -0,0 +1,22 @@
+name: "Link: Microsoft Dynamics 365 form phishing"
+description: |
+  Email body is suspicious, and links to a Microsoft Dynamics form. Known phishing tactic.
+references:
+  - "https://cofense.com/blog/compromised-microsoft-dynamic-365-customer-voice-account-used-for-phishing-attack/"
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and any(ml.nlu_classifier(body.plain.raw).intents,
+      .name != "benign"
+  )
+  and any(body.links, 
+      beta.linkanalysis(.).effective_url.domain.domain == "customervoice.microsoft.com"
```

Comment:
i'd worry a little bit that we'd be invoking LA a bit too much w/o additional signals

also i think we can short-circuit some of this if we first look for the shortened version in the body (as an OR probably)